samples/drivers/lora/receive/sample.yaml

```diff
@@ -7,3 +7,4 @@ tests:
    sample.driver.lora.receive:
      platform_allow: 96b_wistrio
      platform_allow: rak4631_nrf52840
+     platform_allow: nucleo_wl55jc
```

Comment:
In most cases there is one `platform_allow` entry with multiple space separated platforms. Is the method used here (one `platform_allow` entry for each platform) work correctly?

Reply:
Hey you are right, it does seem like in only ran for the last one! The files changed had multiple entries already, assumed it was fine. I'll fix those.

Reply:
That worked, current buildkite run has all the platforms, thanks for spotting the problem. Had a quick look at the other files around the repository and could not find any other instance of this, so we should be good now.